### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by Megha-Pathak**
Preview of landing page:

image

### CODE REVIEW

**Review by sks-21 — APPROVED** [review]
The code looks great!
Just one suggestion, it would be nice if you could add some comments.
You could also deploy it making reviews easy for further components.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,135 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins:200,300,400,500,600,700,800,900&display=swap");
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
```
Comment (sks-21) [inline style.css:7]: separate out common styles in another CSS file.

Comment (sks-21) [inline style.css:7]: separate out common styles in another CSS file

File: style.css [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+header {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  padding: 40px 100px;
```
Comment (Raghav888) [inline style.css:13]: Please use rem

File: style.css [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  filter: invert(1);
+  transform: scale(0.5);
+  transition: 0.5s;
+  color: #111;
+}
+.social li a:hover {
+  transform: scale(0.5) translateY(-15px);
+}
+
+@media (max-width: 991px) {
```
Comment (Raghav888) [inline style.css:124]: use em here

**Review by Raghav888 — APPROVED** [review]
Please use rem and em.
em must be used in a media query.
Have consistency of using rem and em.
If value is less than 10px then only use px

**Pull request merged**